src/app/clusters/valve-configuration-and-control-server/app_config_dependent_sources.cmake

```diff
@@ -12,11 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 # This is the equivalent to app_config_dependent_sources.gni
 TARGET_SOURCES(
   ${APP_TARGET}
   PRIVATE
-    "${CLUSTER_DIR}/valve-configuration-and-control-delegate.h"
+    "${CLUSTER_DIR}/CodegenIntegration.cpp"
+)
+
+# These are the things that BUILD.gn dependencies would pull
+TARGET_SOURCES(
+  ${APP_TARGET}
+  PRIVATE
```

Comment:
should we still have the `-delegate.h` in the list for cmake? cmake should contain all of gn bits.